api/src/services/setup/view-indexer.js

```diff
@@ -55,23 +55,21 @@ const getViewsToIndex = async () => {
  * @return {Promise}
  */
 const indexView = async (dbName, ddocId, viewName) => {
-  let viewIndexed = false;
-
+  const indexing = true;
```

Comment:
I don't think this is necessary... just use `while(true)`!

Reply:
Eslint complains :D 

> Yea! Funny enough, there's an eslint rule [no-constant-condition](https://eslint.org/docs/rules/no-constant-condition) which complains when you do while(true). So a variable is still needed to satisfy it, or ignore the rule.

https://github.com/medic/cht-core/pull/7435#discussion_r805928558

I preferred the variable to eslint ignore.

Reply:
What a BS rule... apparently `while(true)` is bad but `for (;;) {}` is ok?!

Reply:
It makes sense for if/else... maybe we should set `checkLoops:false`?